src/configure.rs

```diff
@@ -102,9 +100,72 @@ pub fn print_configuration(use_default: bool) {
                 .collect::<String>()
         );
     }
```

Comment:
Can you ensure the comments related to `$all` only get printed if `paths` is empty or if it contains `format` or `right_format`?